Cargo.toml

```diff
@@ -185,13 +188,13 @@ heim = { version = "0.1.0-beta.3", optional = true, features = ["full"] }
 uom = { version = "0.29.0", optional = true }
 rust_decimal = "1.8.1"
 mongodb = { version = "1.1.1", optional = true }
+anyhow = { version = "1.0.28" }
```

Comment:
We also use `snafu`, should we standardize on one? They seem to overlap significantly.

Reply:
Yeah, that's a good point.

I am intuitively drawn towards the thiserror+anyhow combination (and saw that anyhow was already used in Vector and so used that).

Having said that, snafu and anyhow do differ, whereas snafu and thiserror are more alike.

See also: https://old.reddit.com/r/rust/comments/dfs1zk/2019_q4_error_patterns_snafu_vs_errderive_anyhow/

Given the difference, and given we already had anyhow, and thiserror is contained to the `remap-lang` crate, I'd vote to leave things as is, and either open up a tracking issue to consolidate the libraries where needed, or leave the situation as is, and have people use the one they prefer (what matters most is how we eventually represent errors to the user, and that the errors we _do_ use implement `std::error::Error`.

Reply:
The `snafu` crate is designed for libraries, while `anyhow` is targeted at application usage, so they could potentially coexist. However, from what I am seeing, this use of `anyhow` *is* in a library, for which we would prefer the other.

Reply:
> However, from what I am seeing, this use of `anyhow` _is_ in a library, for which we would prefer the other.

I'm using `anyhow` it in tests, to produce a string representing the chain of errors that occurred.

Reply:
Ok, that's fair and isn't a problem.